core/modules/parsers/wikiparser/wikiparser.js

```diff
@@ -215,6 +215,8 @@ WikiParser.prototype.parsePragmas = function() {
 			// Set the start and end positions of the pragma rule if
 			if (subTree[0].start === undefined) subTree[0].start = start;
 			if (subTree[subTree.length - 1].end === undefined) subTree[subTree.length - 1].end = this.pos;
+			var rule = Object.getPrototypeOf(nextMatch.rule);
+			for (const node of subTree) node.rule = rule;
```

Comment:
For TW  code we still use ES5 spec. So you need to use `var` instead of `const`
If you need to iterate over "own properties" of an object or array, TW has a utility function.
See: [$tw.utils.eache(obj, callback)](https://github.com/Gk0Wk/TiddlyWiki5/blob/4200495055b86e80ced6e1776080f51d4581623f/boot/boot.js#L141)

Reply:
👌

Reply:
Save this as GPTs custom prompt...